development/image-builder/main.go

```diff
@@ -217,30 +213,32 @@ func runBuildJob(o options, vs Variants) error {
 		}
 		fmt.Println("Successfully built image:", strings.Join(destinations, ", "))
 	}
-
-	var errs []error
-	for variant, env := range vs {
-		var variantTags []string
-		for _, tag := range parsedTags {
-			variantTags = append(variantTags, tag+"-"+variant)
-		}
-		destinations := gatherDestinations(repo, o.name, variantTags)
-		fmt.Println("Starting build for image: ", strings.Join(destinations, ", "))
-		// (@Ressetkk): When variants provided, build doesn't use env files.
-		// Similar logic should be provided once variants are fixed.
-		if err := runFunc(o, variant, destinations, o.platforms, env); err != nil {
-			errs = append(errs, fmt.Errorf("job %s ended with error: %w", variant, err))
-			fmt.Printf("Job '%s' ended with error: %s.\n", variant, err)
-		} else {
-			err := signImages(&o, destinations)
-			if err != nil {
-				return fmt.Errorf("sign encountered error: %w", err)
-			}
-			fmt.Println("Successfully built image:", strings.Join(destinations, ", "))
-			fmt.Printf("Job '%s' finished successfully.\n", variant)
-		}
-	}
-	return errutil.NewAggregate(errs)
+	// TODO (@Ressetkk): Refactor this once someone starts fixing building variants.yaml
+	return fmt.Errorf("building variants is not supported at this moment")
+
+	//var errs []error
```

Comment:
Please remove not used commented code.

Reply:
I left it on purpose, because this bug is tracked in #5975 this implementation might come handy in the future.

Reply:
It is bad practise to leave commented code as you can get it easily from Github history.